conf/portal.application.conf

```diff
@@ -144,6 +144,19 @@ ebean.default = ["models.ebean.*"]
 #ALTER DEFAULT PRIVILEGES IN SCHEMA portal GRANT ALL ON FUNCTIONS TO metrics_app;
 #ALTER DEFAULT PRIVILEGES IN SCHEMA portal GRANT ALL ON TYPES TO metrics_app;
 
+# Cassandra alert repository
+
+play.modules.enabled += "com.arpnetworking.play.PillarModule"
```

Comment:
The module won't do anything unless there are Cassandra databases defined.  We can turn it off if that makes you feel better.  We'd just need to change the testing code to turn it on as well.